bin/lwm2mAgent.js

```diff
@@ -39,6 +39,13 @@ function start() {
         config = require('../config');
```

Comment:
Please add an entry in CHANGES_NEXT_RELEASE about the changes included in this PR, eg:

```
- Add: DTLS support (#xx)
```

where `#xx` is an issue number, if we have some one for DTLS implementation in this repository.

Reply:
(y)